### PR TITLE
build: release 2.10.0 #2073

### LINKED ISSUE #2073: Release 2.10.0
Prepare release 2.10.0

### PR DESCRIPTION
**Pull request by andre-code**
Release 2.10.0

Fix #2073 

/deploy  renku-notebooks=master #persist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+* load notebook preview ([#2052](https://github.com/SwissDataScienceCenter/renku-ui/issues/2052))
+* **server:** use the correct redis namespace ([#2053](https://github.com/SwissDataScienceCenter/renku-ui/issues/2053))
+* show namespace name when there is no owner data, fix not found page size ([#2051](https://github.com/SwissDataScienceCenter/renku-ui/issues/2051))
+
+
+### Features
+
+* save session button ([#1957](https://github.com/SwissDataScienceCenter/renku-ui/issues/1957), [#1985](https://github.com/SwissDataScienceCenter/renku-ui/issues/1985), [#2040](https://github.com/SwissDataScienceCenter/renku-ui/issues/2040), [#1985](https://github.com/SwissDataScienceCenter/renku-ui/issues/1985), [#1957](https://github.com/SwissDataScienceCenter/renku-ui/issues/1957))
+* add environment variables when start a session ([#2058](https://github.com/SwissDataScienceCenter/renku-ui/issues/2058), [#2066](https://github.com/SwissDataScienceCenter/renku-ui/issues/2066))
+* add progress indicator when start a session ([#1879](https://github.com/SwissDataScienceCenter/renku-ui/issues/1879), [#2054](https://github.com/SwissDataScienceCenter/renku-ui/issues/2054))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:15]:
```suggestion
* add progress indicator when starting a session ([#1879](https://github.com/SwissDataScienceCenter/renku-ui/issues/1879), [#2054](https://github.com/SwissDataScienceCenter/renku-ui/issues/2054))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+# [2.10.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.9.0...2.10.0) (2022-10-17)
+
+
+### Bug Fixes
+
+* load notebook preview ([#2052](https://github.com/SwissDataScienceCenter/renku-ui/issues/2052))
+* **server:** use the correct redis namespace ([#2053](https://github.com/SwissDataScienceCenter/renku-ui/issues/2053))
+* show namespace name when there is no owner data, fix not found page size ([#2051](https://github.com/SwissDataScienceCenter/renku-ui/issues/2051))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:8]:
```suggestion
* show namespace name when there is no owner data ([#2051](https://github.com/SwissDataScienceCenter/renku-ui/issues/2051))
* fix not found page size ([#2051](https://github.com/SwissDataScienceCenter/renku-ui/issues/2051))
```

Should we split them into 2 bullet points?
Also, I'm not 100% sure about what this means: `fix not found page size` --> could you rephrase it a bit?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
+# [2.10.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.9.0...2.10.0) (2022-10-17)
+
+
+### Bug Fixes
+
+* load notebook preview ([#2052](https://github.com/SwissDataScienceCenter/renku-ui/issues/2052))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:6]:
```suggestion
* load notebook preview ([#2052](https://github.com/SwissDataScienceCenter/renku-ui/issues/2052))
* prevent sending repeated queries to GitLab graphql ([#2072](https://github.com/SwissDataScienceCenter/renku-ui/issues/2072))
* 
```

I merged another PR to `master` (#2072), here is the additional comment

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Very nice! There is a sentence I would split in 2 that is not totally clear to me, do you mind giving it a look?

P.S. I added another bullet point to the fixes (as a suggestion) because I merged another PR to `master`.

File: CHANGELOG.md (hunk 1 above)
Comment (andre-code) [reply CHANGELOG.md]: thanks, included suggestion!

File: CHANGELOG.md (hunk 3 above)
Comment (andre-code) [reply CHANGELOG.md:6]: ok! thanks

**Review by lorenzo-cavazzi — APPROVED** [review]
:rocket:
